tests/base_test.py

```diff
@@ -1,17 +1,14 @@
+from unittest import TestCase
+from alttester import AltDriver, AltPortForwarding
+from appium import webdriver
 import os
 import sys
 import time
 
 sys.path.append(os.path.dirname(__file__))
```

Comment:
It looks like you did not remove all calls to `sys.path.append(os.path.dirname(__file__))`.